### ISSUE TITLE
RPI 4B

### ISSUE THREAD

**Issue by oelegeirnaert** [reuterbal/photobooth#142]
Hello,

Is their anybody who tried the photobooth on the new RPI4B?
https://www.raspberrypi.org/blog/raspberry-pi-4-on-sale-now-from-35/

I just bought the Odroid N2, but as it is 64bit I do have a lot of issues with modules that are only 64 bit.

Kind Regards,
Oele Geirnaert

**Comment by reuterbal** [comment]
Great question - I don't have a RPI4B, yet. Also, I did not have time to test it on Raspbian buster - I'd be interested to learn whether that causes any issues.

**Comment by oelegeirnaert** [comment]
Yeah, the shop I used before (https://www.sossolutions.nl) will have the RPI4B 4GB on the 2th of july. Once available I'll buy it and test it... I'll keep you posted!

**Comment by SkiTheSlicer** [comment]
I don't have a RPi 4B, but I am running Rasbian Buster (the OS that RPi 4B's require) because I'm trying to work out some issues with qt5 on Rasbian Stretch.

As of right now, the folks over at [piwheels.org](https://blog.piwheels.org/raspberry-pi-4-and-raspbian-buster/) who make the pip packages for ARM haven't build all the latest packages for it. This step will fail: `pip install -e .` with `ERROR: No matching distribution found for opencv-python (from photobooth==0.5.dev3+g40f1cda)`. So, unless you want to build from source, you might need to wait a bit before the RPi 4B can run photobooth.

**Comment by reuterbal** [comment]
That is a good point, thank you for pointing that out!

**Comment by oelegeirnaert** [comment]
Hello,
I recently received my RPI4's and installed Buster.

Everything seems to work fine except 'opencv-python' needs to be removed from the setup.py.
Next you should be able to run the installation with the following command: pip install -e .
So you can not run the photobooth with OpenCV but only with Python-Gphoto2

https://github.com/reuterbal/photobooth/blob/master/setup.py#L187
